### PR TITLE
Maintaining Parity between main and stage

### PR DESCRIPTION
**Pull request by sharmrj**
Sister PR to #41 . So that the same changes are on both main and stage

### CODE REVIEW

File: helix-sitemap.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 1
 
 sitemaps:
   website:
-    origin: https://helpx-internal.stage.adobe.com
+    origin: https://helpx-internal.adobe.com
```
Comment (nkthakur48) [inline helix-sitemap.yaml]: Please revert this change

**Review by nkthakur48 — CHANGES_REQUESTED** [review]

File: helix-sitemap.yaml (hunk 1 above)
Comment (sharmrj) [reply helix-sitemap.yaml]: I ended up using the same branch for both PRs. Shall I revert it after #41 is merged?

**Review by nkthakur48 — DISMISSED** [review]

**Review by adeshmukh-1585 — APPROVED** [review]

**Review by nkthakur48 — APPROVED** [review]
